recipe/meta.yaml

```
@@ -45,8 +45,9 @@ requirements:
     - typing-extensions >=4.5
 
     # Needed by LightningCLI
-    - jsonargparse >=4.27.7
-    - typeshed-client >=2.1
+    - jsonargparse >=4.25.0
```

We can't really control which version of lightning gets picked, so if we assume the latest version, then this becomes:
```suggestion
    - jsonargparse >=4.27.7
```
This might be overly restrictive for older lightning, but usually won't hurt anything and prevent some issues with newer lightning.